### PR TITLE
Filter redux

### PR DESCRIPTION
**Pull request by ethanjurman**
yes

### CODE REVIEW

File: www/js/reducers/filter.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { SET_FILTERS } from '../actions/map-actions';
+
+export function filters(state = {
+  activeFilters:[], openServices:false, hideAlert:false
+}, action) {
+  switch(action.type) {
+    case SET_FILTERS:
+      return action.filters;
```
Comment (stevekroh) [inline www/js/reducers/filter.js]: You need to return a copy: return [ ...action.filters ].

**Comment by stevekroh** [comment]
A few high-level questions:

1. Why does the filter page maintain its own state, and then set all the filters in a big bang on the unmount event?
2. I think we should refactor the `<Marker />` creation code out into a `<MarkerCollection />` or something like that.

File: www/js/components/filter-page.js [hunk 2]
```diff
@@ -106,7 +110,12 @@ class FilterPage extends Component {
       if (this.state.showOptions < this.state.activeFilters.length) {
         func = this.updateFilter.bind(this,this.state.showOptions);
```
Comment (JRJurman) [inline www/js/components/filter-page.js]: this function should have a more descriptive name

File: www/js/components/marker-collection.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React, {Component} from 'react';
```
Comment (JRJurman) [inline www/js/components/marker-collection.js]: why do we have this file?!?

File: www/js/components/point-map.js [hunk 4]
```diff
@@ -93,17 +98,7 @@ class PointMap extends Component {
...
-        <Marker key={alert._id} radius={10} position={alert.location}
-          onclick={() => {
-            if (!this.props.addpoint){
-              dispatch(selectMarker(alert));
-            }
-          }}
-        />
-      );
-    });
+    let alerts = '';
```
Comment (JRJurman) [inline www/js/components/point-map.js]: do we need this variable anymore? or should it be a const?

**Comment by stevekroh** [comment]
:+1:

**Pull request merged**